src/components/TimeseriesPreview/stories/full.md

```diff
@@ -39,7 +39,9 @@ function ExampleComponent(props) {
 | `color`                   | Rendered as a background color for the left side of the component                                          | `string`                                                     | `#6c65ee`                |
 | `dateFormat`              | Defines [date format](https://momentjs.com/docs/#/displaying/format/) to be applied on datapoint timestamp | `string`                                                     | `DD MMM YYYY - HH:mm:ss` |
 | `updateInterval`          | Refresh latest datapoint interval in ms                                                                    | `number`                                                     | `5000`                   |
-| `valueToDisplay`          | Datapoint to be rendered instead of latest datapoint. Pause fetching latest datapoint if provided            | `GetDoubleDatapoint \| GetStringDatapoint`                   |                          |
+| `valueToDisplay`          | Datapoint to be rendered instead of latest datapoint. Pause fetching latest datapoint if provided            | `GetDoubleDatapoint \| GetStringDatapoint`
+| `customName`              | Define a custom name           | `((name?: string) => string) \| string`
+| `customDescription`       | Define a custom description           | `((description?: string) => string) \| string`                   |                          |
```

Comment:
Define a custom description `->` Function that formats timeseries description value to be displayed